bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/api2/Shelly2ApiRpc.java

```diff
@@ -291,8 +298,9 @@ public ShellyDeviceProfile getDeviceProfile(String thingType, @Nullable ShellySe
         }
 
         if (profile.isDimmer) {
-            profile.settings.dimmers = new ArrayList<>();
-            profile.settings.dimmers.add(new ShellySettingsDimmer());
+            ArrayList<@Nullable ShellySettingsDimmer> dimmers = new ArrayList<>();
```

Comment:
```suggestion
            List<@Nullable ShellySettingsDimmer> dimmers = new ArrayList<>();
```

Reply:
Not possible here because it's used for being set in the DTO.